### DISCUSSION TITLE
osu!lazer feels jittery when V-Sync is enabled on macOS

### DISCUSSION THREAD

**Discussion by ThatStella7922** [ppy/osu#17750] [Q&A] [answered]
Basically just title. Whenever I enable V-Sync, the framerate appears to hold a steady 120fps but it feels a lot less consistent and more jittery than when set to 2x.

Also sometimes the framerate will randomly be capped to 80fps when V-Sync is enabled, however I didn’t get to capture it in the video.

Maybe it has to do with the MacBook’s display supporting ProMotion and osu! doesn’t know what to do with it? I’m not sure

macOS 12.3, MacBook Pro 16” (2021, M1 Max)
osu!lazer v2022.409.0

Video showing it:
https://youtu.be/MqIFSDZN5fk

**Comment by peppy** [comment, marked as answer]
vsync on macOS is known to have issues at an SDL level. an issue like this should be tracking it: https://github.com/libsdl-org/SDL/issues/4918

**Comment by ThatStella7922** [comment]
Update, later 2022.1207.0 fixes this completely (presumably with the SDL2 bump to 1.0.630-alpha)

**Reply by peppy** [reply]
Great to hear!
